nestkernel/mpi_manager.cpp

```diff
@@ -87,7 +87,19 @@ nest::MPIManager::init_mpi( int* argc, char** argv[] )
     comm = kernel().music_manager.communicator();
 #else  /* #ifdef HAVE_MUSIC */
     int provided_thread_level;
-    MPI_Init_thread( argc, argv, MPI_THREAD_FUNNELED, &provided_thread_level );
+    int mpi_retcode, mpi_error_len;
+    char* mpi_error_string;
+
+    mpi_retcode = MPI_Init_thread(
+      argc, argv, MPI_THREAD_FUNNELED, &provided_thread_level );
+    if ( mpi_retcode != 0 )
+    {
+      MPI_Error_string( mpi_retcode, mpi_error_string, &mpi_error_len );
+      std::string msg = String::compose(
+        "Error initializing MPI. Error encountered: %1", mpi_error_string );
+
+      LOG( M_ERROR, "MPIManager::init_mpi", msg );
```

Comment:
`LOG( M_ERROR, ...` does not raise an exception. I agree that this should result in an exception like:

```C++
throw KernelException( msg );
```

My other question: does your code actually execute lines 95ff? In #376 it seams as if the code fails within the `MPI_Init_thread` function.